core/src/main/resources/error/error-classes.json

```diff
@@ -551,5 +551,10 @@
       "Writing job aborted"
     ],
     "sqlState" : "40000"
+  },
+  "NULL_COMPARISON_RESULT" : {
+    "message" : [
+      "The comparison result is null. If you want to handle null as 0 (equal), you can set \"<config>\" to \"<value>\"."
```

Comment:
actually, why not hardcode the config name and value here?

Reply:
I just thought it could be reusable. Let me just change to hardcode it for now.